src/main/resources/view/DarkTheme.css

```diff
@@ -123,13 +123,11 @@
 .cell_big_label {
     -fx-font-family: "Segoe UI Semibold";
     -fx-font-size: 16px;
```

Comment:
I think you can add back the text-fill CSS property and it should still work now.